app/src/main/java/org/oppia/app/testing/UrlImageParserTestActivity.kt

```diff
@@ -0,0 +1,17 @@
+package org.oppia.app.testing
+
+import android.os.Bundle
+import android.widget.ImageView
+import org.oppia.app.R
+import org.oppia.app.activity.InjectableAppCompatActivity
+
+/** This is a dummy activity to test [UrlImageParser]. */
+class UrlImageParserTestActivity : InjectableAppCompatActivity() {
```

Comment:
This test activity is not added in manifest or it is unused.

Reply:
Good catch, removed this & the corresponding test. A likely artifact from when I was merging in the pre-finalized upstream changes.